package.json

```diff
@@ -74,7 +74,6 @@
 	},
 	"dependencies": {
 		"@apollo/client": "^3.8.5",
-		"@electron/remote": "^1.1.0",
```

Comment:
This fixes the [Backups add-on fails on load](https://github.com/getflywheel/flywheel-local/pull/1937#issuecomment-1946649159) error by removing `@electron/remote`, the source of the error.